jetty-http/src/main/java/org/eclipse/jetty/http/QuotedQualityCSV.java

```diff
@@ -37,21 +39,16 @@
  */
 public class QuotedQualityCSV extends QuotedCSV implements Iterable<String>
 {    
-    private final static Double ZERO=new Double(0.0);
-    private final static Double ONE=new Double(1.0);
-    
+    private final static Double ZERO = 0.0D;
+    private final static Double ONE = 1.0D;
 
     /**
-     * Function to apply a most specific MIME encoding secondary ordering 
+     * Lambda to apply a most specific MIME encoding secondary ordering
      */
-    public static Function<String, Integer> MOST_SPECIFIC = new Function<String, Integer>()
+    public static Function<String, Integer> MOST_SPECIFIC = s ->
     {
-        @Override
-        public Integer apply(String s)
-        {
-            String[] elements = s.split("/");
-            return 1000000*elements.length+1000*elements[0].length()+elements[elements.length-1].length();
-        }
+        String[] elements = s.split("/");
+        return 1000000*elements.length+1000*elements[0].length()+elements[elements.length-1].length();
```

Comment:
I fail to see what this is doing. Can you add a comment or example?

Reply:
I've updated and now actually use it!